# 先用宽松参数确保能检测到特征
# 逐步收紧参数提高检测质量
# 根据实际凸起大小调整高度阈值
# 根据处理速度需求调整体素大小

feature_extraction_node:
  ros__parameters:
    reference_plane_distance: 0.32  #real
    feature_threshold: 0.005 
    min_feature_height: 0.0         
    max_feature_height: 100.0
    voxel_leaf_size: 0.005             
    statistical_mean_k: 20             
    statistical_stddev_mult: 1.0       
    top_percentage: 5.0             
    min_area_points: 100.0

    # ROI区域参数
    use_roi : true        
    roi_length_ratio: 0.75
    roi_width_ratio: 0.5
    roi_center_x: 0.0
    roi_center_y: 0.0
    roi_max_range_x: 10.0
    roi_max_range_y: 10.0

    #图像处理参数
    image_resolution : 0.01
    morphology_kernel_size :  0.05
    dilation_iterations : 1
    erosion_iterations : 1
    use_dilation_first : true
    debug_display_image : true




